stats/statsaxis.cpp

```diff
@@ -45,31 +70,127 @@ int StatsAxis::guessNumTicks(const QtCharts::QAbstractAxis *axis, const std::vec
 		minSize = minSize * 3 / 2;
 	else
 		minSize *= 2;
-	QRectF chartSize = chart->plotArea();
-	double availableSpace = horizontal ? chartSize.width() : chartSize.height();
-	int numTicks = lrint(availableSpace / minSize);
+	int numTicks = lrint(size / minSize);
```

Comment:
this confuses me - you have `size` as loop local variable in the for loop above (line 62), but which `size` are you accessing here?
Oh, going back I realize that there's a member named `size` now. Then maybe we should rename the variable in the loop above? I know that this is syntactically and semantically valid, but... ugh.

Reply:
Renamed loop variable to `labelSize`.